components/talksPage/talksPage.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { useState } from 'react';
+import style from './talksPage.module.scss';
+import data from '../../assets/videos.json';
+import YouTube from 'react-youtube';
+
+function TalksPage() {
+  const talksList = data;
+  const [currSrc, setSrc] = useState('GHbNaDSWUX8');
+
+  const calcArrowSide = (curr, side, length) => {
+    if (side > curr) {
+      if (curr === length) {
+        return 1;
+      }
+    } else if (side < curr) {
+      if (curr === 1) {
+        return length;
+      }
+    }
+    return side;
+  };
+
+  const onVideoClick = (event) => {
+    event.target.pauseVideo(); // stop chosen video
+    setSrc(event.target.playerInfo.videoData.video_id); // open the chosen video on the big iframe
+  };
+
+  const setVideos = (props) => { // add youtube videos
+    const opts = {
+      height: '200',
+      width: '320',
+    };
+
+    return props.sections.map((video, i) => (
+      <div
+        className={style.item}
+        id={'item' + i + '_' + props.index}
```

Comment:
can we use back ticks? 
`item ${I} ${props.index}`?
also in other places using the + sign?